### ISSUE TITLE
173. ζήω

### ISSUE THREAD

**Issue by jtauber** [jtauber/greek-vocabulary-list#19]
Not in Campbell. Is that because it's (correctly IMO) ζήω not ζάω?

**Comment by seumasjeltzz** [comment]
I think it's just not in Campbell.  
My hunch is that it's less frequent in Campbell's corpus because it's less frequent than other attic-ish alternatives.

**Comment by jtauber** [comment]
ζήω is not in DCC or GE either.

We also have 367. ζάω which DCC and GE _do_ have.

367 is what is aligned with Campbell (1167) ζῆν so Campell does have 173. ζήω, it's just been misaligned.

**Comment by jtauber** [comment]
Although it raises the question why we have BOTH.

**Comment by seumasjeltzz** [comment]
Presumably we should merge them.

**Comment by jtauber** [comment]
yeah, which means the counts need to be combined. I wonder why morpheus would sometimes return one and sometimes return the other.
